website/docs/getting-started/setup/installation-guides/google-cloud-run.mdx

```diff
@@ -269,72 +91,32 @@ This guide is optional, and you can skip to [Create Service](#create-service). F
 
     | Name | Value |
     |------------------------|------------------------------------------| 
-    | **FILESTORE_IP_ADDRESS** | The IP address you noted down in step 4 of [Create a Filestore instance](#create-a-filestore-instance). |
-    | **FILE_SHARE_NAME** | The file share name you noted down in step 4 of [Create a Filestore instance](#create-a-filestore-instance). |
+    | **FILESTORE_IP_ADDRESS** | The IP address you noted down in step for creating a Filestore instance in [Prerequisites]](#prereuisites). |
+    | **FILE_SHARE_NAME** | The file share name you noted down in step for creating a Filestore instance in [Prerequisites]](#prereuisites). |
     | **APPSMITH_MONGODB_URI** | Specify the connection string of the external MongoDB instance required for Appsmith data storage. <br/><br/> <i>Format</i>: <br/>`mongodb://<USERNAME>:<PASSWORD>@<HOST>:<PORT>/?authSource=<DATABASE_NAME>&replicaSet=<REPLICA_SET_NAME>[&options]`|
     | **APPSMITH_ENCRYPTION_SALT** | Specify an encryption salt to encrypt values in the database.|
     | **APPSMITH_ENCRYPTION_PASSWORD** | Specify an encryption password to encrypt values in the database.|
     | **APPSMITH_ENABLE_EMBEDDED_DB** | `0`|
```

Comment:
can we add this in this page. 

we are disabling keycloak by default setting the following env variable

APPSMITH_DISABLE_EMBEDDED_KEYCLOAK | 1 |

in sso page we shall tell them to remove this env variable 
APPSMITH_DISABLE_EMBEDDED_KEYCLOAK | 1 |
and ask them to proceed for sso related steps

Reply:
okay. APPSMITH_DISABLE_EMBEDDED_KEYCLOAK | 1 | to be added instead of 
 | **APPSMITH_ENABLE_EMBEDDED_DB** | `0`| is it?